dev-docs/bidder-adaptor.md

```diff
@@ -495,9 +495,74 @@ if (bid.mediaType === 'video' || (videoMediaType && context !== 'outstream')) {
 
 #### Long-Form Video Content
 
-To support long-form videos it is the responsibility of the adapter to convert their categories into [IAB accepted subcategories]( http://iabtechlab.com/wp-content/uploads/2017/11/IAB_Tech_Lab_Content_Taxonomy_V2_Final_2017-11.xlsx) (links to MS Excel file). Each bid request must return one IAB subcategory.
+{: .alert.alert-info :}
+Following is Prebid's way to setup bid request for long-form. Apadters are free to choose their own approach.
+
+Prebid now accepts multiple bid responses for a single `bidRequest.bids` object. For each Ad pod Prebid expects you to send back n bid responses. It is up to you how you send back bid responses. Prebid's recommendation is that you expand an adpod placement into a set of request objects according to total adpod duration and the range of duration seconds. It also depends on your endpoint as well how you may want to create your request for long-form. Appnexus adapter follows below algorithm to expand its placement. 
```

Comment:
minor typo - should be send back a bid, it reads send back n bid. 

Also minor but is there a standard for Ad pod - should it be Ad Pod, ad pod, or Ad pod? 

I see adpod in the next sentence, if that is referencing the adUnit type we should encase it in backticks `adpod`

I would change this: 
It is up to you how you send back bid responses

To: 
it is up to you how bid responses are returned. 

Minor: Change - according to total adpod duration, to according to the total adpod duration

Reply:
@jsnellbaker Duh on the n ...I didn't notice that. It should be italicized then.